lib/komponent/komponent_helper.rb

```diff
@@ -9,7 +9,11 @@ def component(component, locals = {}, &block)
     require_dependency(component_path)
 
     component_module = "#{component_name}_component".camelize.constantize
-    context = controller.view_context.dup
+    context = self.dup
```

Comment:
This would make the instance variables shared by all components, which is probably not desirable (see #15)